api_create_test.go

```diff
@@ -0,0 +1,56 @@
+package api2go
```

Comment:
I made this it's own test file cause I didn't want there to be a conflict with the upstream repo just for test files